controllers/test-data-controllers.js

```diff
@@ -62,4 +62,18 @@ exports.postCommentForArticle = (req, res, next) => {
     })
     .catch((err) => {
       next(err)});
-};
\ No newline at end of file
+};
+
+exports.partchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+  updateVote(article_id, inc_votes)
+  .then((article) => {
+    if (!article) {
+      return res.status(404).send({ err: 'Article Not Found' });
```

Comment:
You could be checking this inside the model and using `Promise.reject` then handling for the error using error handlers, but this also works